### DISCUSSION TITLE
免費的程式語言學習平台-W3Schools

### DISCUSSION THREAD

**Discussion by HsuChen88** [1101-GS4719-C/Python-Community-GS4719-C#591] [General]
# [W3Schools](https://www.w3schools.com/)
這是一個免費的平台，雖然有付費的版本，但免費的部分已足夠大部分人學習。
![image](https://user-images.githubusercontent.com/99878799/167802768-34f17917-a121-42d3-bdbb-02cf522c4d36.png )
## 可以直接搜尋問題
![image](https://user-images.githubusercontent.com/99878799/167801817-e6faf70e-b474-49a4-91de-eed0b35afdd5.png)

![image](https://user-images.githubusercontent.com/99878799/167804022-81f28604-bbec-4c04-a0c9-683d0db9e662.png)

![image](https://user-images.githubusercontent.com/99878799/167804072-25acf000-b099-4d21-a82f-72ef999e61c8.png)
## 內有豐富教學資源  可以直接選擇一項開始學習
![image](https://user-images.githubusercontent.com/99878799/167804762-da938ff6-d8ef-46bd-83b4-3eff76f4ea4f.png)

![image](https://user-images.githubusercontent.com/99878799/167804786-4a162e46-c428-4c79-9bf1-e924c0201968.png)

![image](https://user-images.githubusercontent.com/99878799/167804811-140e22ce-60bc-4578-8665-1410706d3a13.png)

![image](https://user-images.githubusercontent.com/99878799/167804829-cfeac2c4-b7c8-4c86-8df7-31d894dc8072.png)

![image](https://user-images.githubusercontent.com/99878799/167804860-2ec2e32e-51f1-4010-8ce1-ebe2211158e0.png)

![image](https://user-images.githubusercontent.com/99878799/167804912-ee96efa5-5d13-4bd7-a771-79f1b8ff293b.png)

![image](https://user-images.githubusercontent.com/99878799/167804970-191c05dc-30aa-4213-9059-cb7a8c9274cd.png)

![image](https://user-images.githubusercontent.com/99878799/167805011-6fd67a17-7831-4af1-ad08-713f9c3caec1.png)

## 有內建的程式碼輸入區塊可以直接測試
![image](https://user-images.githubusercontent.com/99878799/167807015-f9883e80-0ec5-4e37-903b-27d9b48cb8b0.png)

## 看完之後還可以做簡易的練習跟測驗
![image](https://user-images.githubusercontent.com/99878799/167807136-fedfb48a-d780-4d03-9cc4-91e6c3850398.png)

### 除了直接查[Stack Overflow](https://stackoverflow.com/)上有沒有類似的問題   若要更完整的說明也可以到[W3Schools](https://www.w3schools.com/)上搜尋~

**Comment by JSHT** [comment]
[CodeCademy](https://www.codecademy.com/) 也是一個很好的選擇

**Comment by 11timyyy** [comment]
好多程式語言，有空可以上去探索看看!

**Comment by handsomeniko** [comment]
感恩 謝謝分享

**Comment by Sazmafuji** [comment]
感謝分享我現在去看看，超讚

**Comment by cordydeng** [comment]
喔喔我也是用這個去自學程式語言，的確滿實用的

**Comment by jcpan821** [comment]
超有用  感謝分享

**Comment by d26488399** [comment]
另一可門課有用到，真的超好用

**Comment by gnsJhenJie** [comment]
W3School的資源還蠻淺顯易懂，而且查詢資料也很方便，可以快速學習到基本的知識。

**Comment by bloodtrail0511** [comment]
其他課程的作業也都是在這網站上找語法的，真的好用

**Comment by stan890425** [comment]
蠻實用的自學程式呢!

**Comment by henrywen547** [comment]
真的很推這個自學工具!
之前就透過這個網站學了一點html跟css
登入後還可以使用自己的空間來練習!

**Comment by awkward-willy** [comment]
W3Schools 真的是相當方便理解且實用的學習平台
不過現在有各式各樣的學習資源
不論是程式書、部落格發文、Youtube 教學影片、Udemy、CodeCademy......
都是能快速增進自我實力的好地方
不過前提還是願意花時間看啦，不能像我一樣三分鐘熱度......

**Comment by minniewei** [comment]
之前用這學過css,javascript,html真的非常好用

**Comment by Lai27330** [comment]
有這麼好用的學習資源能夠自學程式和查詢問題真的很方便，謝謝樓主分享。

**Comment by jason79461385** [comment]
CSS用這個學感覺真的很方便，可以馬上驗收自己會不會

**Comment by liangyinglly** [comment]
這個好物搭配不懂的上網查 真的進步很快

**Comment by Lynn12378** [comment]
真的超好用，還有簡單的填空可以練習

**Comment by Jimmy911218** [comment]
謝謝分享~之後能上上瀏覽看看

**Comment by Hchejxnc** [comment]
記得之前程式課寫網頁的時候，也是使用這個網站去尋找資料，真的幫助很大~

**Comment by 110409538** [comment]
感謝分享,現在網上提供程式語言教學資源的網站越來越多了.
